holoviews/core/spaces.py

```diff
@@ -708,22 +736,22 @@ def __getitem__(self, key):
         for a previously generated key that is still in the cache
         (for one of the 'open' modes)
         """
-        tuple_key = util.wrap_tuple_streams(key, self.kdims, self.streams)
+        # Split key dimensions and data slices
+        if key is Ellipsis:
+            return self
+        elif key == ():
+            map_slice, data_slice = (), ()
+        else:
+            map_slice, data_slice = self._split_index(key)
```

Comment:
``NdMapping.__getitem__`` does this with empty tuples:

```python
elif indexslice == () and not self.kdims:
     return self.data[()]
```

Won't work here unfortunately, but I guess I could still handle it in ``_split_index``, ``NdMapping`` would just never use that path.